purchase_lot/tests/test_purchase_lot.py

```diff
@@ -8,6 +8,7 @@ class TestPurchaseLot(TransactionCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        cls.partner_id = cls.env.ref("base.res_partner_1")
```

Comment:
Is this addition necessary?

Reply:
It simplifies a little bit the tests because it is used in both tests of the module